auk-notebook-text.ipynb

```diff
@@ -133,10 +133,6 @@
     "\n",
     "RESULTS_STEP = 5,\n",
     "\n",
-    "# Change if you want a different filename.\n",
-    "\n",
-    "OUTPUT_FILENAME = \"./filtered_text.txt\", # filename if you want to output to another file.\n",
-    "\n",
```

Comment:
Just a note here that removing this element does not mean that OUTPUT_FILENAME does not have a value, since this variable (actually, `au_notebook.output_filename`) will have a value in the class that can be used.

Reply:
Do you think this is OK to remove then - what do you recommend?

Reply:
Yes.  Fine to remove for this. There may be a consideration for the au_notebook class, as it appears that the function currently does not even use the variable as a default value. If OUTPUT_FILENAME is not set, nb.output_filename is "./filtered_text.txt".  I say it's fine to be hidden in the shadows for now.

Reply:
OK sounds good, thanks @greebie !